README.md

```diff
@@ -60,9 +63,39 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+yum -y install epel-release
+yum -y install \
```

Comment:
I guess in the future we should move this to their own shell scripts so that we can share them with the container specs to keep them in sync...